CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Release Hightlights
 
 ## Important Notes
+- [#335] The session expiry for the OIDC provider is now taken from the Token Response (expires_in) rather than from the id_token (exp)
```

Comment:
this appears unrelated to this pull-request, maybe there was some merge mixup

Reply:
Thanks. 
Yes, a mixup with a merge.